Package.swift

```diff
@@ -51,7 +51,6 @@ var package = Package(
     .package(url: "https://github.com/airbnb/lottie-ios", from: "3.1.9"),
     .package(url: "https://github.com/jrendel/SwiftKeychainWrapper", from: "4.0.1"),
     .package(url: "https://github.com/SDWebImage/SDWebImage", exact: "5.10.3"),
-    .package(url: "https://github.com/SDWebImage/SDWebImageSVGNativeCoder", from: "0.1.1"),
     .package(url: "https://github.com/SDWebImage/SDWebImageSwiftUI", from: "2.2.0"),
```

Comment:
Unrelated but is this still needed?

Reply:
still used to display NFT that has gif format image